bioblend/galaxy/__init__.py

```diff
@@ -106,6 +109,8 @@ def __init__(
         self.tool_data = tool_data.ToolDataClient(self)
         self.folders = folders.FoldersClient(self)
         self.tool_dependencies = tool_dependencies.ToolDependenciesClient(self)
+        if session is not None:
+            self.session = session
```

Comment:
I think you should pass `session` to the parent constructor above instead: `super().__init__(url, key, email, password, verify=verify, session=session)`